src/main/java/io/debezium/connector/yugabytedb/transforms/StriimCompatible.java

```diff
@@ -0,0 +1,208 @@
+package io.debezium.connector.yugabytedb.transforms;
+
+import java.util.*;
+
+import io.debezium.connector.yugabytedb.transforms.SchemaUtil;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Schema.Type;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yb.util.Pair;
+
+public class StriimCompatible<R extends ConnectRecord<R>> implements Transformation<R> {
```

Comment:
Add a java doc comment explaining the use of this transformer in brief. For reference, see other classes in the repo.

Reply:
done